notebooks/introduction/categorical_distribution.ipynb

```diff
@@ -0,0 +1,656 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    import optax
</pre><p>We can adapt to same try: except: pattern as we use for pml books.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/code-first-ml/book1/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>